### ISSUE TITLE
关于batch normalization层

### ISSUE THREAD

**Issue by huyi1989** [watsonyanghx/CNN_LSTM_CTC_Tensorflow#18]
如果想要保存跟新moving_mean和moving_variance的话，好像要写以下代码：
extra_update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)
with tf.control_dependencies(extra_update_ops):
    train_op = optimizer.minimize(loss)
但是好像没找到相关内容，想问作者是不是漏了写了？

**Comment by watsonyanghx** [comment]
这里设置了`updates_collections=None`，可以看一下[tf.contrib.layers.batch_norm](https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm)。
